.gitignore

```diff
@@ -5,6 +5,7 @@ pkg
 *.patch
 *.gem
 *.sh
+.idea
```

Comment:
This belongs in your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).